src/client/skins.js

```diff
@@ -0,0 +1,76 @@
+import * as THREE from "three";
+import { renderCore } from "./render/render-core";
+import { levelManager } from "./level-manager";
+import { marbleManager } from "./marble-manager";
+import { cameras } from "./render/cameras";
+import domReady from "./dom-ready";
+
+// Set up core (rendering, level & marble management)
+renderCore.initialize(cameras.CAMERA_FREE);
+renderCore.updateCallback = function(deltaTime) {
+	levelManager.activeLevel.update(deltaTime);
+
+	if (_rotateMarbles) {
+		_leftMesh.rotation.x += .01;
+		_leftMesh.rotation.y += .01;
+		_leftMesh.rotation.z += .01;
+
+		_rightMesh.rotation.x += .01;
+		_rightMesh.rotation.y += .01;
+		_rightMesh.rotation.z += .01;
+	}
+};
+levelManager.initialize();
+marbleManager.initialize();
+
+// Add marbles
+let _rotateMarbles = false;
+let _marble = {
+	skinId: "default",
+	color: "#ffffff"
+};
+_marble.geometry = new THREE.SphereBufferGeometry(1, 32, 32);
```

Comment:
At a glace, I think it would be better to use `marbleManager.spawnMarble()` here and let that return the mesh. If we manage the preview marble through the same methods as the client, we'll know for sure they show up the same way. If anything changes in there (e.g. materials or maybe even marble topology), we won't have to edit it here as well.

Reply:
This is a good idea, I'll pour it into an issue and fix it once I get to redoing the skins! I don't think it's worth changing right now though.